src/azure-cli/azure/cli/command_modules/acs/_format.py

```diff
@@ -86,14 +87,12 @@ def find_preview_versions(versions_bag):
             if upgrade.get('isPreview', False):
                 preview[upgrade['kubernetesVersion']] = True
     find_preview_versions(result.get('controlPlaneProfile', {}))
-    find_preview_versions(result.get('agentPoolProfiles', [{}])[0])
```

Comment:
why this removed?

Reply:
we moved agentpool upgrade profile information to a new api and will implement a new command 'az aks nodepool get-upgrades'.